mri/parallel_mri/extract_sensitivity_maps.py

```diff
@@ -91,55 +92,39 @@ def extract_k_space_center_and_locations(data_values, samples_locations,
     return data_thresholded, center_locations
 
 
-def gridding_2d(points, values, img_shape, method='linear', point_min=None,
-                point_max=None):
+def grided_FT_ND(points, values, grid, method):
     """
-    Interpolate non-Cartesian data into a cartesian grid
+    This function calculates the grided fourier transform
```

Comment:
It doesn't look like this function computes a grided fourier transform, but a grided inverse fourier transform.
Therefore I would advise changing the name to something like `grided_inverse_fourier_transform_nd`.